x-pack/plugins/enterprise_search/server/lib/get_oauth_token_package_params.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ENTERPRISE_SEARCH_KIBANA_COOKIE } from '../../common/constants';
+
+export function getOAuthTokenPackageParams(cookieHeaders: string | string[] | undefined) {
```

Comment:
syntax nit - let's prefer arrow functions for consistency across the codebase (unless we really need `this` for a specific reason)

```suggestion
export const getOAuthTokenPackageParams = (cookieHeaders: string | string[] | undefined) => {
```